inyoka_theme_default/templates/ikhaya/event_suggest.html

```diff
@@ -0,0 +1,35 @@
+{#
+    ikhaya/event_suggest.html
+    ~~~~~~~~~~~~~~~~~~~~
+
+    This page is for new calendar events.
+
+    :copyright: (c) 2007-2016 by the Inyoka Team, see AUTHORS for more details.
+    :license: BSD, see LICENSE for more details.
+#}
+{% extends 'ikhaya/base.html' %}
+
+{% set styles = ['editor', 'datetimefield'] %}
+{% set scripts = ['WikiEditor','DateTime'] %}
+
+{% block breadcrumb %}
+  {{ macros.breadcrumb_item(_('Events'), href('ikhaya', 'events')) }}
+  {{ macros.breadcrumb_item( _('Suggest event') ) }}
+{% endblock %}
+
+{% block content %}
+  <h2>{% trans %}Suggest event{% endtrans %}</h2>
+  <p>
+    {% trans link_calendar=href('portal', 'calendar'), link_newsteam=href('wiki', 'ubuntuusers/Ikhayateam', _anchor='Teammitglieder') %}
+      By using this form you can suggest a new entry for the <a href="{{ link_calendar }}">calendar</a>. A team member will review your suggestion as soon as possible. If you want to change an existing calendar entry please contact the <a href="{{ link_newsteam }}">news team</a>.
+    {% endtrans %}
+  </p>
+
+  {{ macros.outer_form(csrf_token(), form) }}
+
```

Comment:
I have added a ticket for the form-macro
https://github.com/inyokaproject/theme-default/issues/194